File: Phase - 9 Automation Testing Using Python/POMWithoutPF/pages/home_page.py
from selenium.webdriver.common.by import By

class HomePage:
    my_account_icon = (By.XPATH, "//i[@class = \"fa fa-user\"]")
    login_button = (By.LINK_TEXT, "Login")
    search_input = (By.XPATH, "//input[@name = 'search']")
    search_button = (By.XPATH, "//i[@class = \"fa fa-search\"]")
    

File: Phase - 7 Automation Testing Using Java/Assessment-3/cucumber/src/test/java/pages/HomePage.java
package pages;

import org.openqa.selenium.By;

public class HomePage {
	
	public By searchInput = By.xpath("//input[@name = \"search\"]");
	public By productsNames = By.xpath("//div[@class = \"product-thumb\"]/div[2]/div/h4");
	public By AddressBook = By.xpath("//a[text() = \"Address Book\"]");
	public By myAccount = By.xpath("//i[@class = \"fa fa-user\"]");
	public By loginLink = By.xpath("//a[@href = \"https://tutorialsninja.com/demo/index.php?route=account/login\"]");
}
